src/components/Forms/Forms.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import SwitchForms from "./FormsSwitch";
+import FoodForms from "../FoodForm/FoodForms";
+import WaterForms from "../WaterForm/WaterForms";
+import MoodForm from "../MoodForm/MoodForm";
+import SleepForm from "../SleepForm/SleepForm";
+
+interface IForms {
+  id: number;
+}
+
+const Forms: React.FC<IForms> = ({ id }) => {
+  return (
+    <SwitchForms id={id}>
```

Comment:
maybe step is a better word than id, and also let's create enum for this

Reply:
You mean enum for step? For example. 
food: 1,
water: 2, etc.

Reply:
yeah

Reply:
it can be `FORM_TYPE { FOOD: 'food', WATER: 'water', ... }`

Reply:
Are you sure you want to switch to string instead of number? Then you have to change state and rewrite data-id.

Reply:
yeah, it makes more sense, also it should be `data-value` instead of `data-id`